sdk/typescript/src/account/account.ts

```diff
@@ -197,13 +197,13 @@ export class Account implements IAccount {
 
   async querySessionKeys(cursor: Bytes | null, limit: number): Promise<IPage<ISessionKey>> {
     const accessPath = `/resource/${this.address}/0x3::session_key::SessionKeys`
-    const state = await this.provider.getStates(accessPath)
+    const state = await this.provider.getState(accessPath)
```

Comment:
But `getState` here. We should unify them. I suggest use `getStates`, keep consistent with RPC